crates/api_models/src/admin.rs

```diff
@@ -201,6 +209,10 @@ pub struct MerchantAccountResponse {
     ///Default business details for connector routing
     #[schema(value_type = Vec<PrimaryBusinessDetails>)]
     pub primary_business_details: Vec<PrimaryBusinessDetails>,
+
+    ///Will be used to expire client secret after certain amount of time to be supplied in seconds
+    ///(900) for 15 mins
+    pub intent_fulfillment_time: Option<i64>,
```

Comment:
change this type also to `u32`

Reply:
We can't do this as we are requesting unsigned value in request and internally converting it to i64 hence we are responding with the i64 itself.